timezonechange/addeventlistener-timezonechange-fired.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
```

Comment:
nit: remove `<html>`, `<body>`, `</body>`, `</html>` like in the other tests in this patch

Reply:
why?

Reply:
It's simpler (less redundant & unrelated code to worry about while reading the tests) and more consistent with existing tests. The Google HTML Style Guide recommends this as well (not that it applies here): https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags